netlify.toml

```diff
@@ -1,3 +1,4 @@
 [build]
     functions = "packages/web/functions"
     command = "yarn build"
+    publish = "packages/web/dist/elvia-designsystem/browser"
```

Comment:
The new application builder moved the output path (added the `/browser` to the end) of the angular app, so I just added it here instead of making changes in the Netlify settings in the browser.